styleguide/js/zoolander.js

```diff
@@ -21,14 +21,52 @@ jQuery(document).ready(function ($) {
     jQuery ('.title a').click(function (event) {
       jQuery (this).parent().parent().find('.fa-arrow-right').animate({  borderSpacing: +90 }, {
       step: function(now,fx) {
-          $(this).css('-webkit-transform','rotate('+now+'deg)'); 
+          $(this).css('-webkit-transform','rotate('+now+'deg)');
           $(this).css('-moz-transform','rotate('+now+'deg)');
           $(this).css('transform','rotate('+now+'deg)');
           counter = 1;
       },
       duration:'slow'
       },'linear');
     });
-});
 
+    /*brought in from www rackspace.js*/
+    var Rackspace = window.Rackspace || {};
+    function isMobile() {
+      return typeof window.matchMedia !== 'undefined' ? window.matchMedia("only screen and (max-width: 768px)").matches : false;
+    }
+    //Additional viewport check (replacing isMobile)
+    function isDesktop() {
+      return typeof window.matchMedia !== 'undefined' ? window.matchMedia("only screen and (min-width: 992px)").matches : false;
+    }
+    /**
+     * Simple accordion logic for use on mobile displays
+     */
+    Rackspace.Accordion = (function($) {
```

Comment:
This is the first instance of custom JS that is in Zoolander that will be brought into www. I would prefer not to copy/paste it into www and I don't think the RJL is a good place for it, so we should look at putting this into it's own file in using a `script` element to bring it in prior to this `ready` block. Once it is in it's own file we can integrate it into www with gulp and add the `Rackspace.Accordion.init();` line in `rackspace.js`.